platform/core/PolyPodCoreAndroid/core/src/test/java/coop/polypoly/core/ExampleUnitTest.kt

```diff
@@ -0,0 +1,16 @@
+package coop.polypoly.core
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
```

Comment:
Sorry, what's the point of this?

Reply:
This is just an example file generated by Idea it appears - let's just remove it for now I'd say. Adding some smoke tests for the bindings would definitely be useful though.

Reply:
Yeah, it could be removed. As for the tests for bindings, I still have to think to which extent we want to test the interaction with `core` on different platforms. I would not want to have duplicated tests between platforms.